pkg/console/util.go

```diff
@@ -79,6 +80,35 @@ func validatePingServerURL(url string) error {
 	})
 }
 
+func validateNTPServers(ntpServerList []string) error {
+	if len(ntpServerList) == 0 {
+		return fmt.Errorf("NTP Server should not be empty.")
+	}
```

Comment:
Should it be optional? We tend to show optional inputs after the required ones so that users can easily skip them

Reply:
It's optional now and I put it after the password panel. Thank you.